sympy/integrals/transforms.py

```diff
@@ -1284,6 +1285,10 @@ def _fourier_transform(f, x, k, a, b, name, simplify=True):
     For suitable choice of a and b, this reduces to the standard Fourier
     and inverse Fourier transforms.
     """
+
+    if limit(f, x, -oo) != 0 and (limit(f, x, -oo) is not S.NaN or limit(f, x, -oo).is_number or limit(f, x, -oo) == S.Infinity or limit(f, x, -oo) == S.NegativeInfinity):
```

Comment:
Yaa, sure I'll do it after I find the correct conditions to be used in if statement